app/models/contact.rb

```diff
@@ -1,5 +1,13 @@
 class Contact < ApplicationRecord
-  validates :first_name, :last_name, presence: true
-  
   belongs_to :user
+  belongs_to :company, optional: true
+
+  VALID_PHONE_REGEX = /\A\d{3}\-\d{3}-\d{4}\z/
+  VALID_EMAIL_REGEX = /\A[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}\z/
```

Comment:
I was looking forward to incorporating a regex into a project, very thorough and will set our app apart in terms of thoroughness. Looks like a fun code spike to explore.